charts/tidb-cluster/values.yaml

```diff
@@ -77,6 +77,7 @@ pd:
   # we can only specify clusterIP and loadBalancerIP now
   service:
     clusterIP: "None"
+    portName: "client"
```

Comment:
what if we don't set `portName` by default in helm value. will the original name in the code be used? if it is, I think we can omit it in the values.

Reply:
It will use original name in the code. But do we need to remind users of this feature.

Reply:
we can give an example in the value files, like this:

```
service:
    clusterIP: "None"
    # portName: "<the default name will be used>"
```

what do you think?

Reply:
I agree, i will follow this.

Reply:
the absent field in yaml will be parsed as `nil` in go struct. this is backward compatible but we also provide a way for users to configure this field.

Reply:
cc @Yisaer what do you think?

Reply:
yes, you re right.